### PR TITLE
Add groups report to admin dashboard

### PR DESCRIPTION
**Pull request by finitud**
This ticks the first two items in https://trello.com/c/iO3YJnyW/191-staff-dashboard-groups-report

### CODE REVIEW

File: h/admin.py [hunk 1]
```diff
@@ -207,6 +207,12 @@ def badge_remove(request):
     request.db.delete(models.Blocklist.get_by_uri(uri))
     return badge_index(request)
 
+@view.view_config(route_name='admin_groups',
+                  request_method='GET',
+                  renderer='h:templates/admin/groups.html.jinja2',
+                  permission='admin_groups')
+def groups_index(request):
+    return {"groups": models.Group.all(), "request": request}
```
Comment (nickstenning) [inline h/admin.py]: You shouldn't need to pass `request` in explicitly, as it's one of the [system values provided to renderers by Pyramid](http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/templates.html#renderer-system-values).

**Comment by nickstenning** [comment]
This looks great. Only one minor point about template context, and perhaps you could just add a sentence to the top of the page to explain its purpose (see the other admin pages for examples)?

**Comment by nickstenning** [comment]
Oh, and one more thought: perhaps instead of separate columns for the username and user email, it might make sense to have one column which links the creator's username to `/admin/users?username={{ group.creator.username }}`?

File: h/admin.py (hunk 1 above)
Comment (finitud) [inline h/admin.py]: Oh, thanks, will fix! :+1:

**Comment by nickstenning** [comment]
LGTM! :cake:

**Pull request merged**